packages/rum-vue/test/specs/index.spec.js

```diff
@@ -196,7 +196,10 @@ describe('APM route hooks', () => {
     await router.isReady()
     router.afterEach(() => {
       const tr = apm.getCurrentTransaction()
-      expect(tr.spans[0].name).toBe('before-route-enter')
```

Comment:
This test was failing in the pipeline but not in the macbook. Debugging this in another laptop (M1 it's too much powerful 😄 )  helped me to understand where the problem was:

Since we are adding the 1 second delay, we cannot assure anymore that the first span will be the one expected. 

Karma during its lifecycle may trigger long tasks,  because of that the `spans[0]` was not the expected and it was the longtask instead

 Screenshot with longtask logged during test execution:

![longtask-screenshot](https://user-images.githubusercontent.com/15065076/162280060-6f0e02ae-53c1-4bff-83a6-4a8c8c7b0d16.png)

`context.html` is the value of the src of Karma's iframe